Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Article.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Article
+    {
+        public int ArticleId { get; set; }
+
+        public string Name { get; set; }
+        public string FileLocation { get; set; }
+        public string CoverImageLocation { get; set; }
+        public string Description { get; set; }
+        public bool InRotation { get; set; }
+        public string Nation { get; set; }
+        public ICollection<ArticlesRead> articlesReads { get; set; }
```

Comment:
here you're using camelCase but for the rest PascalCase, please use one standard